### PR TITLE
[improvement](profile)Add workload group in audit log and profile

### PR DESCRIPTION
**Pull request by wangbo**
## Proposed changes

Add workload group in audit log and profile.

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/common/profile/SummaryProfile.java [hunk 1]
```diff
@@ -72,7 +73,7 @@ public class SummaryProfile {
     // a column, so that should not
     // add many columns here. Add to ExcecutionSummary list.
     public static final ImmutableList<String> SUMMARY_KEYS = ImmutableList.of(PROFILE_ID, TASK_TYPE,
-            START_TIME, END_TIME, TOTAL_TIME, TASK_STATE, USER, DEFAULT_DB, SQL_STATEMENT);
+            START_TIME, END_TIME, TOTAL_TIME, TASK_STATE, USER, DEFAULT_DB, WORKLOAD_GROUP, SQL_STATEMENT);
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/common/profile/SummaryProfile.java:76]: 不要放在这里，这里会显示在fe ui的表格里。
需要放到EXECUTION_SUMMARY_KEYS

File: fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java [hunk 2]
```diff
@@ -301,7 +301,8 @@ private void auditAfterExec(String origStmt, StatementBase parsedStmt,
                 .setStmtId(ctx.getStmtId())
                 .setQueryId(ctx.queryId() == null ? "NaN" : DebugUtil.printId(ctx.queryId()))
                 .setTraceId(spanContext.isValid() ? spanContext.getTraceId() : "")
-                .setFuzzyVariables(!printFuzzyVariables ? "" : ctx.getSessionVariable().printFuzzyVariables());
+                .setFuzzyVariables(!printFuzzyVariables ? "" : ctx.getSessionVariable().printFuzzyVariables())
+                .setWorkloadGroup(ctx.getWorkloadGroupName());
```
Comment (yiguolei) [inline fe/fe-core/src/main/java/org/apache/doris/qe/ConnectProcessor.java:305]: .setWorkloadGroup(ctx.getWorkloadGroupName());
then  call set fuzzy variables

**Review by HappenLee — DISMISSED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Comment by yiguolei** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.25 seconds
 stream load tsv:          533 seconds loaded 74807831229 Bytes, about 133 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17162175816 Bytes

**Review by gavinchou — APPROVED** [review]

**Pull request merged**